docs/concepts/packages.md

```diff
@@ -108,7 +108,9 @@ all types that its package installs, as well as `Secrets`, `ConfigMaps`, and
 
 > Note that the Crossplane RBAC manager can be configured to reject permissions
 > for certain API groups. If a package requests permissions that Crossplane is
-> configured to reject, the package will fail to be installed. 
+> configured to reject, the package will fail to be installed.
+> Authorized permissions should be added to the rbac manager provider-clusterrole
+> (the cluster role defined by the provider-clusterrole flag in the rbac manager)
```

Comment:
Thanks so much for adding the note of clarity here @fahedouch! One of the nice features of Kubernetes RBAC is [`ClusterRole` aggregation](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles), which we employ with the allowed permissions for providers, by using a label:
https://github.com/crossplane/crossplane/blob/2cb935ac62650d2932e73c4e2957ac55f02bd3af/cluster/charts/crossplane/templates/rbac-manager-allowed-provider-permissions.yaml#L13

I believe this is our recommended strategy for adding allowed permissions for providers, rather than mutating the `ClusterRole` we create directly. Would you mind updating the note to reflect that?

Reply:
@hasheddan WDYT ?

Reply:
@fahedouch looks good! Although it looks like there may be an unintended build submodule update included now?

Reply:
Sorry @hasheddan . fixed